assets/component-accordion.css

```diff
@@ -49,6 +49,8 @@
 .accordion__content {
   margin-bottom: 1.5rem;
   word-break: break-word;
+  overflow-x: auto;
+  padding: 0 0.6rem;
```

Comment:
Add to add the padding to account for the focus ring. As adding the `overflow` property was cutting it off as noticed by Meli.